api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt

```diff
@@ -185,22 +183,8 @@ class WebSecurityConfig(
                     }
                 )
             )
-            .authenticated()
+            .permitAll()
     }
-        .and().httpBasic().run {
-            val userFromProps = configurationProperties.basicCredentials?.split(' ')?.run {
```

Comment:
do we still need corresponding field `basicCredentials` in `ConfigurationProperties`?

Reply:
No, removed